### PR TITLE
celery:update celery to 4.4

### PR DESCRIPTION
**Pull request by pazembrz**
lock elasticsearch client to 7.1.x

INSPIR-3044

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -14,11 +14,12 @@ babel = "==2.*,>=2.0.0,>=2.5.1"
 backoff = "==1.*,>=1.0.0,>=1.4.3"
 "backports.tempfile" = ">=1.0rc1"
 beard = "==0.*,>=0.0.0,>=0.2.0"
-celery = "<4.2.0,==4.*,>=4.0.0,>=4.1.0"
+celery = "<4.5.0,==4.*,>=4.4.0"
 click = "<7.0,==6.*,>=6.7.0"
 click-spinner = "==0.*,>=0.0.0,>=0.1.8"
 coverage = {version = "==4.*,>=4.0.0,>=4.5.4", optional = true}
 docutils = {version = "<0.16,==0.*,>=0.15.0,>=0.15.2", optional = true}
+elasticsearch = "<=7.2.0"
```
Comment (drjova) [inline pyproject.toml]: isn't it dangerous? shall we add ```>=7.1.0,<7.2.0```

Comment (pazembrz) [reply pyproject.toml]: elasticsearch major version is managed by invenio-search.
`invenio-search = {version = "~=1.2.3,<1.3.0", extras = [ "elasticsearch7",]}`

File: poetry.lock [hunk 2]
```diff
@@ -271,39 +271,48 @@ category = "main"
...
 memcache = ["pylibmc"]
-msgpack = ["msgpack-python (>=0.3.0)"]
+mongodb = ["pymongo (>=3.3.0)"]
+msgpack = ["msgpack"]
 pymemcache = ["python-memcached"]
 pyro = ["pyro4"]
-redis = ["redis (>=2.10.5)"]
+redis = ["redis (>=3.2.0)"]
 riak = ["riak (>=2.0)"]
+s3 = ["boto3 (>=1.9.125)"]
```
Comment (drjova) [inline poetry.lock]: ?

File: pyproject.toml [hunk 3]
```diff
@@ -49,7 +49,7 @@ invenio-admin = "==1.1.2"
 invenio-assets = "1.1.3"
 invenio-base = "==1.2.0"
 invenio-cache = "==1.0.0"
-invenio-celery = "==1.0.0"
+invenio-celery = "==1.1.0"
```
Comment (drjova) [inline pyproject.toml]: why not ``1.1.1``?

Comment (pazembrz) [reply pyproject.toml]: 1.1.1 Locks celery on 4.2.x https://github.com/inveniosoftware/invenio-celery/commit/593b6647a948cf081b8efb42272f8613b9e98ef4

File: poetry.lock (hunk 2 above)
Comment (pazembrz) [reply poetry.lock]: It's generated automatically, It's not put in pyproject.toml, just like django ;)

File: pyproject.toml (hunk 3 above)
Comment (drjova) [reply pyproject.toml]: :(

**Pull request merged**